test/test_backend.py

```diff
@@ -170,6 +170,77 @@ def test_counts_over_time(self):
                     # Part 2: all other row counts should be non-zero
                     self.assertGreater(df[col][row], 0, msg=f"Table '{row}' had 0 rows in run '{col}'")
 
+    def test_cr_hierarchy_related_csets(self):
```

Comment:
@hugwuoke This looks like a great start! Looks good so far the way you've set it up, and I like how things are well commented.

Reply:
FYI, just did a tiny lil bit of project management; added this and [the issue](https://github.com/jhu-bids/TermHub/issues/355) to the project board, added labels, and connected the issue to the PR via the 'development' menu on the right, so it should close automatically when this PR is complete. If for any reason the PR doesn't fully address the issue, you can go to that 'development' menu and un-connect the issue.